docs/cloud-soar/mssp.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: mssp
-title: Cloud SOAR for MSSPs
+title: MSSP
```

Comment:
For SEO purposes, I would do:

`title` Cloud SOAR for MSSPs
`sidebar_label` MSSPs

Reply:
Yep! I had them backwards.